### ISSUE TITLE
[Bug]: setting speed limit causes client hang.

### ISSUE THREAD

**Issue by zBeeble42** [nextcloud/desktop#5242]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

Briefly, the client is locking up, repeatably.  Locking up includes not responding to an open from the dock, locking up context menus in the windows shell and only responding to a task-manager kill.  If the window is open, it locks up showing with "0B of 4.1MB file 1 of 34" or "7B of 4.1MB file 1 of 34"

Pertinent points:
-  I started with an old version of the desktop client (version lost).  I upgraded to 3.6.7 (?) and then to a daily release 3.6.50 (grabbed 2022-12-06).  All versions exhibit the same behaviour.
- Nextcloud is up-to-date.  No changes there (is in production).
- if I add files to the local store, those files are uploaded, then this bug manifests.
- It was stopping on one file in the local root (of the nextcloud) folder. "langs.xml" I didn't need that file, so I removed it.  No change (found this with the client logs).
- I also found all files renamed with _conflict_ in them.  I resolved those conflicts.  No change.
- The client logs stop mostly on the same file, but sometimes a different file.

Extra diagnostic points (not my first bug report):
- computer rebooted, disk CHKDSK'd.  client restarted multiple times.  Behaviour is repeatable
- logs created, but an awful lot of non-opaque information in them.  Would prefer to share privately _or_ to cut out small sections.
- no other users of same nextcloud affected.
- no errors in nextcloud server log
- I tried removing files the log stopped on.  Did not help.
- I tried removing all the .db files in the root ... causing a complete resync.  Did not help.

### Steps to reproduce

Start nextcloud, wait.  Obviously, not for you.  This is 100% repeatable on my system.  Nextcloud is effectively not working until fixed.  I'm not sure what's causing it.

### Expected behavior

Nextcloud doesn't lock up?

### Which files are affected by this bug

langs.xml

### Operating system

Windows

### Which version of the operating system you are running.

Windows 10 workstation

### Package

Appimage

### Nextcloud Server version

25.0.1

### Nextcloud Desktop Client version

3.6.50 (although also any others I tested)

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell
Debug	news	new parser added : FeedIo\Standard\Rdf		2022-12-06T15:13:42-0500
Debug	news	new parser added : FeedIo\Standard\Rss		2022-12-06T15:13:42-0500
Debug	news	new parser added : FeedIo\Standard\Atom		2022-12-06T15:13:42-0500
Debug	news	new parser added : FeedIo\Standard\Json		2022-12-06T15:13:42-0500
Debug	logreader	Could not find resource file "/apps/logreader/css/../js/logreader-main.css"		2022-12-06T15:13:41-0500
Debug	spreed	Controller OCA\Talk\Controller\SignalingController::pullMessages created 65 QueryBuilder objects, please check if they are created inside a loop by accident.		2022-12-06T15:13:41-0500
Debug	spreed	Controller OCA\Talk\Controller\SignalingController::pullMessages created 65 QueryBuilder objects, please check if they are created inside a loop by accident.		2022-12-06T15:13:38-0500
```

### Additional info

[nextcloud-short.log](https://github.com/nextcloud/desktop/files/10170051/nextcloud-short.log)

**Comment by zBeeble42** [comment]
Just saying that I chopped the bottom of the client log... and sanitized it.  Feel free to ask for more, but I'd prefer that to be somewhat private.

**Comment by zBeeble42** [comment]
to test that this wasn't a filesystem thing, I ran the following in L4W (Linux for windows) in the root of the nextcloud directory:

find . -type f -ls -exec cat "{}" >/dev/null \;

... which completed without error.

**Comment by Nordiii** [comment]
Similar with me, but Nextcloud goes completely unresponsive. I can’t open anything, and it just says “syncing” but without using bandwidth.

Some part of the Log:

```
2022-12-08 09:12:44:554 [ info nextcloud.sync.engine C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\syncengine.cpp:689 ]:	#### Reconcile (aboutToPropagate) ####################################################  547 ms
2022-12-08 09:12:44:554 [ debug nextcloud.sync.statustracker C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\syncfilestatustracker.cpp:229 ]	[ OCC::SyncFileStatusTracker::slotAboutToPropagate ]:	Investigating "afolder/afolder_prakt6_afile.txt" OCC::SyncFileItem::NoStatus CSyncEnums::CSYNC_INSTRUCTION_SYNC OCC::SyncFileItem::Up
2022-12-08 09:12:44:554 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 1 -4722185707584817330
2022-12-08 09:12:44:554 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:295 ]	[ OCC::SqlQuery::exec ]:	SQL exec "SELECT path, inode, modtime, type, md5, fileid, remotePerm, filesize,  ignoredChildrenRemote, contentchecksumtype.name || ':' || contentChecksum, e2eMangledName, isE2eEncrypted,   lock, lockOwnerDisplayName, lockOwnerId, lockType, lockOwnerEditor, lockTime, lockTimeout  FROM metadata  LEFT JOIN checksumtype as contentchecksumtype ON metadata.contentChecksumTypeId == contentchecksumtype.id WHERE phash=?1"
2022-12-08 09:12:44:554 [ debug nextcloud.gui.socketapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\socketapi\socketapi.cpp:205 ]	[ OCC::SocketListener::sendMessage ]:	Sending SocketAPI message --> "STATUS:SYNC:C:\\Users\\name\\Nextcloud\\afolder" to QLocalSocket(0x1c805d0b9a0)
2022-12-08 09:12:44:554 [ debug nextcloud.gui.socketapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\socketapi\socketapi.cpp:205 ]	[ OCC::SocketListener::sendMessage ]:	Sending SocketAPI message --> "STATUS:SYNC:C:\\Users\\name\\Nextcloud" to QLocalSocket(0x1c805d0b9a0)
2022-12-08 09:12:44:554 [ debug nextcloud.sync.statustracker C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\syncfilestatustracker.cpp:229 ]	[ OCC::SyncFileStatusTracker::slotAboutToPropagate ]:	Investigating "afolder/afolder_prakt7_afile.txt" OCC::SyncFileItem::NoStatus CSyncEnums::CSYNC_INSTRUCTION_NEW OCC::SyncFileItem::Up
2022-12-08 09:12:44:554 [ debug nextcloud.gui.socketapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\socketapi\socketapi.cpp:205 ]	[ OCC::SocketListener::sendMessage ]:	Sending SocketAPI message --> "STATUS:IGNORE:C:\\Users\\name\\Nextcloud\\.owncloudsync.log" to QLocalSocket(0x1c805d0b9a0)
2022-12-08 09:12:44:554 [ debug nextcloud.gui.socketapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\socketapi\socketapi.cpp:205 ]	[ OCC::SocketListener::sendMessage ]:	Sending SocketAPI message --> "STATUS:IGNORE:C:\\Users\\name\\Nextcloud\\.sync_1b0a808c8923.db" to QLocalSocket(0x1c805d0b9a0)
2022-12-08 09:12:44:554 [ debug nextcloud.gui.socketapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\socketapi\socketapi.cpp:205 ]	[ OCC::SocketListener::sendMessage ]:	Sending SocketAPI message --> "STATUS:IGNORE:C:\\Users\\name\\Nextcloud\\.sync_1b0a808c8923.db-shm" to QLocalSocket(0x1c805d0b9a0)
2022-12-08 09:12:44:554 [ debug nextcloud.gui.socketapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\socketapi\socketapi.cpp:205 ]	[ OCC::SocketListener::sendMessage ]:	Sending SocketAPI message --> "STATUS:IGNORE:C:\\Users\\name\\Nextcloud\\.sync_1b0a808c8923.db-wal" to QLocalSocket(0x1c805d0b9a0)
2022-12-08 09:12:44:555 [ debug nextcloud.gui.socketapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\socketapi\socketapi.cpp:205 ]	[ OCC::SocketListener::sendMessage ]:	Sending SocketAPI message --> "STATUS:IGNORE:C:\\Users\\name\\Nextcloud\\Desktop.ini" to QLocalSocket(0x1c805d0b9a0)
2022-12-08 09:12:44:555 [ info nextcloud.sync.engine C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\syncengine.cpp:696 ]:	#### Reconcile (aboutToPropagate OK) ####################################################  548 ms
2022-12-08 09:12:44:555 [ debug nextcloud.sync.database C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\syncjournaldb.cpp:2418 ]	[ OCC::SyncJournalDb::commitInternal ]:	Transaction commit "post treewalk" and starting new transaction
2022-12-08 09:12:44:556 [ info nextcloud.sync.engine C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\syncengine.cpp:803 ]:	Network Limits (down/up)  -75 0
2022-12-08 09:12:44:556 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:295 ]	[ OCC::SqlQuery::exec ]:	SQL exec "SELECT tmpfile, etag, errorcount, path FROM downloadinfo"
2022-12-08 09:12:44:556 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:295 ]	[ OCC::SqlQuery::exec ]:	SQL exec "SELECT path,transferid FROM uploadinfo"
2022-12-08 09:12:44:556 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:295 ]	[ OCC::SqlQuery::exec ]:	SQL exec "SELECT path FROM blacklist"
2022-12-08 09:12:44:556 [ debug nextcloud.sync.database C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\syncjournaldb.cpp:2418 ]	[ OCC::SyncJournalDb::commitInternal ]:	Transaction commit "post stale entry removal" and starting new transaction
2022-12-08 09:12:44:556 [ debug nextcloud.gui.socketapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\socketapi\socketapi.cpp:205 ]	[ OCC::SocketListener::sendMessage ]:	Sending SocketAPI message --> "STATUS:SYNC:C:\\Users\\name\\Nextcloud" to QLocalSocket(0x1c805d0b9a0)
2022-12-08 09:12:44:556 [ info nextcloud.sync.engine C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\syncengine.cpp:750 ]:	#### Post-Reconcile end ####################################################  549 ms
2022-12-08 09:12:44:556 [ info nextcloud.sync.bandwidthmanager C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\bandwidthmanager.cpp:342 ]:	Upload Bandwidth limit changed 0 -75
2022-12-08 09:12:44:556 [ info nextcloud.gui.folder C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\folder.cpp:950 ]:	#### Propagation start ####################################################
2022-12-08 09:12:44:556 [ debug nextcloud.gui.socketapi C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\socketapi\socketapi.cpp:485 ]	[ OCC::SocketApi::slotUpdateFolderView ]:	Not sending UPDATE_VIEW for "1" because status() is OCC::SyncResult::SyncRunning
2022-12-08 09:12:44:559 [ info nextcloud.gui.application C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\gui\owncloudgui.cpp:198 ]:	Sync state changed for folder  "https://www.example.de/remote.php/dav/files/NextCloudUser/" :  "Sync running"
2022-12-08 09:12:44:559 [ info nextcloud.sync.propagator.root.directory C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\owncloudpropagator.cpp:1285 ]:	scheduleSelfOrChild OCC::PropagatorJob::NotYetStarted pending uploads 0 subjobs state OCC::PropagatorJob::NotYetStarted
2022-12-08 09:12:44:559 [ warning nextcloud.sync.propagator.directory C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\owncloudpropagator.cpp:1049 ]:	Useless task found for file "afolder/afolder_prakt6_afile.txt" instruction CSyncEnums::CSYNC_INSTRUCTION_SYNC
2022-12-08 09:12:44:559 [ warning nextcloud.sync.propagator.directory C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\owncloudpropagator.cpp:1049 ]:	Useless task found for file "afolder/afolder_prakt7_afile.txt" instruction CSyncEnums::CSYNC_INSTRUCTION_NEW
2022-12-08 09:12:44:560 [ info nextcloud.sync.propagator.root.directory C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\owncloudpropagator.cpp:1309 ]:	OCC::SyncFileItem::Success slotSubJobsFinished OCC::PropagatorJob::Running pending uploads 2 subjobs state OCC::PropagatorJob::Finished
2022-12-08 09:12:44:560 [ info nextcloud.sync.propagator C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\owncloudpropagator.cpp:1341 ]:	PropagateRootDirectory::scheduleDelayedJobs
2022-12-08 09:12:44:561 [ info nextcloud.sync.checksums C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\checksums.cpp:230 ]:	Computing "MD5" checksum of "C:/Users/name/Nextcloud/afolder/afolder_prakt6_afile.txt" in a thread
2022-12-08 09:12:44:561 [ info nextcloud.sync.checksums C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\checksums.cpp:230 ]:	Computing "MD5" checksum of "C:/Users/name/Nextcloud/afolder/afolder_prakt7_afile.txt" in a thread
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 1 "afolder/afolder_prakt6_afile.txt"
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 2 0
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 3 0
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 4 0
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 5 4030
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 6 1669835332
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:151 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 7 "MD5:a3af7b05f0c8ab1faa17381a05a24cd4"
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:295 ]	[ OCC::SqlQuery::exec ]:	SQL exec "INSERT OR REPLACE INTO uploadinfo (path, chunk, transferid, errorcount, size, modtime, contentChecksum) VALUES ( ?1 , ?2, ?3 , ?4 ,  ?5, ?6 , ?7 )"
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:328 ]	[ OCC::SqlQuery::exec ]:	Last exec affected 1 rows.
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\syncjournaldb.cpp:2418 ]	[ OCC::SyncJournalDb::commitInternal ]:	Transaction commit "Upload info" and starting new transaction
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:151 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 1 "afolder/afolder_prakt6_afile.txt"
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:295 ]	[ OCC::SqlQuery::exec ]:	SQL exec "SELECT baseFileId, baseModtime, baseEtag, basePath FROM conflicts WHERE path=?1;"
2022-12-08 09:12:44:562 [ info nextcloud.sync.propagator.bulkupload C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\bulkpropagatorjob.cpp:183 ]:	"/afolder/afolder_prakt6_afile.txt" transmission checksum "a3af7b05f0c8ab1faa17381a05a24cd4" "C:/Users/name/Nextcloud/afolder/afolder_prakt6_afile.txt"
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 1 "afolder/afolder_prakt7_afile.txt"
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 2 0
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 3 0
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 4 0
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 5 1763
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:145 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 6 1670453986
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:151 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 7 "MD5:29d14ad8a4ab4c6e0771a6d33c38ca21"
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:295 ]	[ OCC::SqlQuery::exec ]:	SQL exec "INSERT OR REPLACE INTO uploadinfo (path, chunk, transferid, errorcount, size, modtime, contentChecksum) VALUES ( ?1 , ?2, ?3 , ?4 ,  ?5, ?6 , ?7 )"
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:328 ]	[ OCC::SqlQuery::exec ]:	Last exec affected 1 rows.
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\syncjournaldb.cpp:2418 ]	[ OCC::SyncJournalDb::commitInternal ]:	Transaction commit "Upload info" and starting new transaction
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common/ownsql.h:151 ]	[ OCC::SqlQuery::bindValue ]:	SQL bind 1 "afolder/afolder_prakt7_afile.txt"
2022-12-08 09:12:44:562 [ debug nextcloud.sync.database.sql C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\common\ownsql.cpp:295 ]	[ OCC::SqlQuery::exec ]:	SQL exec "SELECT baseFileId, baseModtime, baseEtag, basePath FROM conflicts WHERE path=?1;"
2022-12-08 09:12:44:562 [ info nextcloud.sync.propagator.bulkupload C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\bulkpropagatorjob.cpp:183 ]:	"/afolder/afolder_prakt7_afile.txt" transmission checksum "29d14ad8a4ab4c6e0771a6d33c38ca21" "C:/Users/name/Nextcloud/afolder/afolder_prakt7_afile.txt"
2022-12-08 09:12:44:563 [ info nextcloud.sync.accessmanager C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\accessmanager.cpp:78 ]:	4 "" "https://www.example.de/remote.php/dav/bulk" has X-Request-ID "021479ec-fc80-4493-85ce-7d89eccd22fe"
2022-12-08 09:12:44:563 [ debug nextcloud.sync.cookiejar C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\cookiejar.cpp:90 ]	[ OCC::CookieJar::cookiesForUrl ]:	QUrl("https://www.example.de/remote.php/dav/bulk") requests: (QNetworkCookie("__Host-nc_sameSiteCookielax=true; secure; HttpOnly; expires=Fri, 31-Dec-2100 23:59:59 GMT; domain=www.example.de; path=/"), QNetworkCookie("__Host-nc_sameSiteCookiestrict=true; secure; HttpOnly; expires=Fri, 31-Dec-2100 23:59:59 GMT; domain=www.example.de; path=/"), QNetworkCookie("oc_sessionPassphrase=MyPassPhrase; secure; HttpOnly; domain=www.example.de; path=/"), QNetworkCookie("MyCookie; secure; HttpOnly; domain=www.example.de; path=/"))
2022-12-08 09:12:44:563 [ info nextcloud.sync.networkjob C:\Users\sysadmin\AppData\Local\Temp\2\windows-12296\client-building\desktop\src\libsync\abstractnetworkjob.cpp:361 ]:	OCC::PutMultiFileJob created for "https://www.example.de" + "" "OCC::BulkPropagatorJob"

```

**Comment by joe-eklund** [comment]
I also just started running into this issue. :( Same problem on both 3.6.2 and 3.6.4.

**Comment by Obi-Wahn** [comment]
Same problem with different Nextcloud client versions and windows 11.

**Comment by tomdereub** [comment]
Hi, I found a workaround here : https://github.com/nextcloud/desktop/issues/4106
Removing speed limits has solved the problem in my case (windows 10, desktop 3.6.4, server 25.02). Then I put speed limits again and it seems it's working normally...

**Comment by Nordiii** [comment]
Yes deleting this block from the nextcloud.cfg indeed fixed the issue for me:
```
[BWLimit]
downloadLimit=80
uploadLimit=10
useDownloadLimit=0
useUploadLimit=-1
```

**Comment by Obi-Wahn** [comment]
Yes, no speed limits fixed the issue for me too.

**Comment by zBeeble42** [comment]
OP here ... so this is a workaround ... but not an acceptable one.  Needs to be fixed.

I was able to start the client and just turn off the limit --- in my case, getting to the sync failure takes most of a minute.  So I suppose doing it this way depends on your setup.

**Comment by tomdereub** [comment]
I got the problem with all computers of my company (30 persons), and with some others linked to my personal server. Seems to be related to the update to nextcloud 25, the bug appears with various versions of nextcloud desktop. It is systematic if speed limits has been set.
The fact that it blocks windows explorer is really annoying, and it's hard to debug if you don't find this issue on github. People try unsuccessfully to re-install nextcloud, to restart synchronisation from scratch (with risk of data loss for non-synced files), etc...
Even if there is a workaround, I think it's a major issue that should be quickly solved. I still see the "0. Needs triage" label, I think it's time to approve this bug and switch to "1. To develop".

**Comment by joe-eklund** [comment]
Well looks like the issue has returned for me. Seems like it only worked for about a week and now I'm stuck on preparing for sync again.

**Comment by skytux** [comment]
Looks similar than my issue.

System & client version:
`################## "Nextcloud" locale: "en_150" ui_lang: QVariant(Invalid) version: "3.6.4stable-Win64 (build 20221208)" os: "Windows 10 Version 2009"`

Conditions:
- [x] when bandwidth limit activated
- [x] added a PDF file with some contents (203,536 bytes) to nextcloud folder through explorer.exe
- [x] same with 1 byte long txt file
- [x] same with 46,101 bytes long PDF file
- [x] same with 287,837 bytes long PNG file
- [x] same with 544,231 bytes long JPG file 
- [x] same with 999,999 bytes binary file 
- [ ] does not occurs with empty txt file
- [ ] does not occurs with 12,728,592 bytes binary .IMG file
- [ ] does not occurs with 2,524,275 bytes PDF file
- [ ] does not occurs with 1,112,777 bytes PDF file
- [ ] does not occurs with 1,048,576 bytes binary file
- [ ] does not occurs with 1,000,000 bytes binary file

**So it seems to happen for files between 1 byte to 999,999 bytes.**

I tried to trace the bug, using some debug tools:

- I add a PDF file with some contents (203,536 bytes).
- The file gets read by a nextcloud.exe child thread (log + procmon)
- procmon shows it by chunks of 16,384 bytes until EOF, but tries to read more from it several times. + queries some metadata.
> ![image](https://user-images.githubusercontent.com/3109677/210195231-5f9dd632-65c1-4d1a-8c81-9acc935a4734.png)
- MD5 hash is computed according to the log file.
- nextcloud.exe child thread opens the file one last time (without any further operation with it). Stack trace is:
![image](https://user-images.githubusercontent.com/3109677/210196353-17fd8cee-92c2-47ac-9f8f-c82e8ee7880e.png)

Last log messages before it freezes:
`[ OCC::CookieJar::cookiesForUrl ]:	QUrl("[...]/remote.php/dav/bulk") requests: (QNetworkCookie [...]
OCC::PutMultiFileJob created for "https://[...]" + "" "OCC::BulkPropagatorJob"`
(libsync\abstractnetworkjob.cpp:361)

- 950 bytes data is sent through TCP to NextCloud server (seen by procmon.  - Not my server - I don't have logs for it)
- Now nextcloud.exe UI is frozen, eating 100% CPU of one core
- **explorer.exe UI also gets frozen!**

- Now, if I delete the previously added file, UI stays frozen and eating CPU, but 1 new line appears in log file:
- `[ debug nextcloud.gui.folderwatcher C:\Users\sysadmin\AppData\Local\Temp\2\windows-12824\client-building\desktop\src\gui\folderwatcher_win.cpp:160 ]	[ OCC::WatcherThread::watchChanges ]:	Skipping syncing of "C:/path_to_folder_where_is_the_tested_file`

**Comment by skytux** [comment]
The issue seems to dissapear when disabling the Upload bandwidth limit.
Download might be left be on. But might be a problem to retrieve small files. Can't test easily now.

I imagine something like the bandwidth limiting feature not working well in bulk mode - and/or with small files.

**Comment by stesee** [comment]
> Yes deleting this block from the nextcloud.cfg indeed fixed the issue for me:
> 
> ```
> [BWLimit]
> downloadLimit=80
> uploadLimit=10
> useDownloadLimit=0
> useUploadLimit=-1
> ```

Adding the Path of the file: %APPDATA%\Nextcloud\nextcloud.cfg

**Comment by nfrawley93** [comment]
@stesee a faster way on is actually just turning off Wifi so Nextcloud never gets to the "sync" part. Then you can still open the GUI and turn off throttling. (I needed a slightly simpler way for some of my users to understand)

You still need to kill the client in task manager first though to un-freeze it.

- Kill Nextcloud client
- Turn of Wifi or Ethernet (manually or software)
- Start Nextcloud and turn off throttling bandwidth
- Turn on Wifi or Ethernet

What an frustrating issue. I have throttling on so I don't over saturate my network every time a large file is uploaded. 

Can't believe this isn't a higher priority.

**Comment by Tublick** [comment]
had the same issue but updating the Windowsclient to 3.6.6 (from 3.4.0) had solved the problem for me.

**Comment by mlazzarotto** [comment]
Still happening on **3.7.3**. Nextcloud clients uses a lot of cpu and the computer begins overheating. Left-clicking or right-clicking on the tray icon does nothing, and the tray icon is the 'sync in progress' one.

**Comment by Ericchi** [comment]
I have the same exact problem on MacOS. Tried every possible fix until I remembered that I have recently set an upload limit. Removing it fixed the issue. Please, fix it. It's really important to me because I have a poor connection that can't keep up and is easily saturated whenever I upload a file unless I put an upload limit.

**Comment by silen72** [comment]
Still happening on 3.8.0. Tested on Windows 10. Besides being completely unresponsive, it also seems to add significant lag to using the windows explorer.

**Comment by claucambra** [comment]
Hi folks, this is now fixed with #5680
